doc/developer/guide-testing.md

```diff
@@ -257,11 +259,65 @@ provide most of the coverage, but it's worth adding some (*very*) basic
 testdrive tests (e.g., `> SELECT DATE '1999-01-01'`) to ensure that our pgwire
 implementation is properly serializing dates.
 
-Like the [Unit Tests](#unitintegration-tests), many testdrive tests require
-Zookeeper, Kafka, and the Confluent Schema Registry.
+#### Usage via mzcompose/Docker
```

Comment:
Let's just say "Usage via mzcompose" and let the fact that mzcompose is Docker nonsense be documented over in the mzcompose documentation.